plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/FieldSetterFactory.java

```diff
@@ -59,8 +64,17 @@
 import static io.trino.spi.type.VarbinaryType.VARBINARY;
 import static java.lang.Float.intBitsToFloat;
 import static java.lang.Math.toIntExact;
+import static java.util.Collections.unmodifiableList;
+import static java.util.Collections.unmodifiableMap;
 import static java.util.Objects.requireNonNull;
 
+/**
+ * Class that generates field setters for Hive data.
```

Comment:
This doesn't add more information than the class name already carries -- it's a Factory for FieldSetter, and we're in Hive module.

remove

Reply:
Removed. (This was just a header line for the rest of the Javadoc.)